app/reducers/ui.js

```diff
@@ -35,6 +35,9 @@ export default function auth(state = initialState, action) {
     case 'FETCHING_UPDATED_TASK_INFORMATION_SUCCESS': {
       return Object.assign({}, state, { showTaskDetailsSidebarLoading: false });
     }
+    case 'TOGGLE_SWIMLANE': {
+      return document.querySelector('section[data-reactid*="'+action.payload.id+'"]').classList.toggle('swimlane--collapsed');
```

Comment:
Reducers aren't intended to update the UI in this way, what would be better is if each swimlane had a `visible` option which then the `TOGGLE_SWIMLANE` case would update the value accordingly. Then in `Swimlane/index.jsx`you'd check the `visible` value for the prop and add the class there.

For future reference reducers should only be used to update the state tree. http://redux.js.org/docs/basics/Reducers.html